bindings/experimental/distrdf/python/DistRDF/PythonMergeables.py

```diff
@@ -48,3 +49,7 @@ def GetValue(self, backend: BaseBackend):
             snapshot_chain.Add(filename)
         # Create a new rdf with the chain and return that to user
         return backend.make_dataframe(snapshot_chain)
+
+
+# A type alias to signify any type of result that can be returned from the RDataFrame API
+RDataFrameFutureResult = Union[ROOT.RDF.RResultPtr, ROOT.RDF.Experimental.RResultMap, SnapshotResult, AsNumpyResult]
```

Comment:
not for this PR, but maybe `AsNumpyResult` should live in this module and be imported by `_pythonization._rdataframe` instead?

Reply:
Unclear, `AsNumpyResult` is actually tied to the `AsNumpy` feature, before being used in `DistRDF`. Moving it here would mean making that part of PyROOT depend on a class defined in a `DistRDF` submodule. One alternative is splitting the class in two. The actual result of `AsNumpy` would still live only in PyROOT, whereas in `DistRDF` I could create a second class and call it `AsNumpyFuture` or similar, that wraps the `AsNumpyResult` from PyROOT and then adds the `Merge` functionality on top